### ISSUE TITLE
Vue App option doesn't seem to work

### ISSUE THREAD

**Issue by jakob-e** [jxnblk/ram#17]
![image](https://user-images.githubusercontent.com/1699461/41148943-90e5b1e6-6b0a-11e8-9018-28e3d54888a4.png)

RAM-0.0.9.dmg

**Comment by elliotsa** [comment]
Yeah I noticed that as well. I submitted a pull request here to fix it #14
